plyer/platforms/linux/storagepath.py

```diff
@@ -44,7 +44,7 @@ def _get_root_dir(self):
         return "/"
 
     def _get_documents_dir(self):
-        directory = self._get_from_user_dirs("DOCUMENT")
+        directory = self._get_from_user_dirs("DOCUMENTS")

```

Comment:
This is right, according to https://cgit.freedesktop.org/xdg/xdg-user-dirs/tree/user-dirs.defaults.